src/test/scala/chiselTests/MultiAssign.scala

```diff
@@ -5,7 +5,7 @@ package chiselTests
 import chisel3._
 import chisel3.testers.BasicTester
 import chisel3.util._
-import circt.stage.ChiselStage
+import _root_.circt.stage.ChiselStage
```

Comment:
Obviously it's fine to specify starting at the root, but alternatively you could put this import before the `import chisel3.util._` to avoid ambiguity with the new package you're adding.